BRANCH_TODO.md

```diff
@@ -0,0 +1,45 @@
+Dependencies:
+
+- [x] libuv
+- [-] lua
+- [ ] luajit
+- [x] MSGPACK_URL https://github.com/msgpack/msgpack-c/archive/c-6.0.1.tar.gz
+- [x] UNIBILIUM_URL https://github.com/neovim/unibilium/archive/d72c3598e7ac5d1ebf86ee268b8b4ed95c0fa628.tar.gz
+- [ ] LIBVTERM_URL https://github.com/neovim/libvterm/archive/v0.3.3.tar.gz
+- [-] luv
+- [-] lpeg
+- [x] compat53 (c api part only - but likely enough)
+- [x] TREESITTER_URL https://github.com/tree-sitter/tree-sitter/archive/v0.22.5.tar.gz
+
+non-glibc:
+- [ ] GETTEXT_URL https://github.com/neovim/deps/raw/b9bf36eb31f27e8136d907da38fa23518927737e/opt/gettext-0.20.1.tar.gz
+- [ ] LIBICONV_URL https://github.com/neovim/deps/raw/b9bf36eb31f27e8136d907da38fa23518927737e/opt/libiconv-1.17.tar.gz
+
+Runtime dependencies:
+
+- [ ] treesitter parsers
```

Comment:
Those shouldn't be an issue, they compile fine with zig (in fact, that is _the_ preferred method on Windows...) For the record, here's the build flags from nvim-treesitter:
```
'zig',
      'cc',
      '-o',
      'parser.so',
      repo.files, // `src/parser.c`, `src/scanner.c` if it exists
      '-lc',
      '-Isrc',
      '-shared',
      '-Os',
```
on non-Win, add `-Wall -Wextra -fPIC -Werror=implicit-function-declaration`